File: app/src/main/java/com/owncloud/android/ui/preview/PreviewImageErrorFragment.java
package com.owncloud.android.ui.preview;

import android.os.Bundle;
import android.view.LayoutInflater;
import android.view.View;
import android.view.ViewGroup;

import com.owncloud.android.R;
import com.owncloud.android.ui.fragment.FileFragment;

import androidx.annotation.Nullable;

import static com.owncloud.android.ui.activity.FileActivity.EXTRA_FILE;

/**
 * A fragment showing an error message
 */

public class PreviewImageErrorFragment extends FileFragment {
    @Nullable
    @Override
    public View onCreateView(LayoutInflater inflater, @Nullable ViewGroup container, @Nullable Bundle savedInstanceState) {
        return inflater.inflate(R.layout.preview_image_error_fragment, container, false);
    }

    public static FileFragment newInstance() {
        FileFragment fileFragment = new PreviewImageErrorFragment();
        Bundle bundle = new Bundle();

        bundle.putParcelable(EXTRA_FILE, null);
        fileFragment.setArguments(bundle);

        return fileFragment;
    }
}

File: app/src/main/java/com/owncloud/android/ui/preview/PreviewImageErrorFragment.kt
/*
 * Nextcloud - Android Client
 *
 * SPDX-FileCopyrightText: 2018 Tobias Kaminsky <tobias@kaminsky.me>
 * SPDX-License-Identifier: AGPL-3.0-or-later OR GPL-2.0-only
 */
package com.owncloud.android.ui.preview

import android.os.Bundle
import android.view.LayoutInflater
import android.view.View
import android.view.ViewGroup
import com.owncloud.android.R
import com.owncloud.android.ui.activity.FileActivity
import com.owncloud.android.ui.fragment.FileFragment

/**
 * A fragment showing an error message.
 */
class PreviewImageErrorFragment : FileFragment() {
    override fun onCreateView(inflater: LayoutInflater, container: ViewGroup?, savedInstanceState: Bundle?): View? {
        return inflater.inflate(R.layout.preview_image_error_fragment, container, false)
    }

    companion object {
        fun newInstance(): FileFragment {
            val fileFragment: FileFragment = PreviewImageErrorFragment()
            val bundle = Bundle()

            bundle.putParcelable(FileActivity.EXTRA_FILE, null)
            fileFragment.arguments = bundle

            return fileFragment
        }
    }
}
